config/engines.yml

```diff
@@ -191,6 +191,16 @@ radon:
     - \.py$
   default_ratings_paths:
     - "**.py"
+reek:
+  image: codeclimate/codeclimate-reek
+  description: "Reek examines Ruby classes, modules, and methods and reports any code smells it finds."
+  community: true
+  upgrade_languages:
+    - Ruby
```

Comment:
Is it worth it to set the upgrade and enable patterns if it's not going to be enabled by default?